.github/workflows/container.reusable.yml

```diff
@@ -0,0 +1,91 @@
+name: Build container image
+on:
+  workflow_call:
+    inputs:
+      name:
+        description: Name of the image to build
+        required: true
+        type: string
+      tag:
+        description: Tag of the image to build
+        required: true
+        type: string
+      load:
+        description: Whether to load the image after building it
+        default: false
+        required: false
+        type: boolean
+      push:
+        description: Whether to push the image after building it
+        default: false
+        required: false
+        type: boolean
+      latest:
+        description: Whether to also build (and push, if push == true) the `latest` tag
+        default: false
+        required: false
+        type: boolean
+      artifact:
+        description: Name of the artifact where the binaries have been uploaded
+        required: false
+        default: nri-kubernetes
+        type: string
+      platforms:
+        description: Comma-separated platforms to build for
+        required: false
+        default: linux/amd64,linux/arm64,linux/arm
+        type: string
+
+jobs:
+  container:
+    name: Build container image
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Download all artifacts from build job
+        uses: actions/download-artifact@v2
+        with:
+          name: ${{ inputs.artifact }}
+          path: bin
+
+      # It is common for branch names to have / characters, which cannot be used as container tags.
+      - name: Sanitize tag
+        id: tag
+        run: |
+          tag="${{ inputs.tag }}"
+          echo "::set-output name=name::${tag//\//-}"
+
+      - name: Build container images
+        run: |
+          load=""
+          if [[ "${{ inputs.load}}" == "true" ]]; then
+            load="--load"
+          fi
+
+          docker buildx build $load --platform=${{ inputs.platforms }} \
+            -t ${{ inputs.name }}:${{ steps.tag.outputs.name }} \
+            .
+
+      - uses: docker/login-action@v1
+        if: ${{ inputs.push }}
+        with:
+          username: ${{ secrets.FSI_DOCKERHUB_USERNAME }}
+          password: ${{ secrets.FSI_DOCKERHUB_TOKEN }}
```

Comment:
Would we add these two as an `input` too?

Reply:
Definitely, good point!